read.go

```diff
@@ -166,6 +169,20 @@ func readPixelData(r dicomio.Reader, t tag.Tag, vr string, vl uint32, d *Dataset
 		return nil, errors.New("the Dataset context cannot be nil in order to read Native PixelData")
 	}
 
+	if r.SkipPixelData() {
```

Comment:
can we add a test for this in read_test.go? It can be a similar pattern to TestReadNativeFrames, just that with the option set we expect there to be no data parsed. 

See my comment below, but perhaps we can just test at the readPixelData level (as this behavior should be the same for native and encapsulated pixeldata)

Link:
https://github.com/suyashkumar/dicom/blob/836cc53a9bb7a481e1acf91e53a9ca4fde1149f4/read_test.go#L216